prime-router/pom.xml

```diff
@@ -36,9 +36,9 @@
         <!-- Define skip flags with common defaults for developers. -->
         <package.skip.azure>false</package.skip.azure>
         <package.skip.cli>true</package.skip.cli>
-        <flyway.user>prime</flyway.user>
-        <flyway.password>changeIT!</flyway.password>
-        <flyway.url>jdbc:postgresql://localhost:5432/prime_data_hub</flyway.url>
+        <pg.user>prime</pg.user>
+        <pg.password>changeIT!</pg.password>
+        <pg.url>jdbc:postgresql://localhost:5432/prime_data_hub</pg.url>
```

Comment:
It seemed reasonable to change these since they are no longer used for just Flyway. Nothing else that is checked in to the repo uses them, but I don’t know if this will affect other scripts or ops-related code outside this repo.